packages/react-components/src/molecules/__tests__/SocialIcons.test.tsx

```diff
@@ -53,3 +54,19 @@ it('renders icon and link', () => {
     ]
   `);
 });
+
+it('does not contain content when there are no social icons', () => {
+  const { container } = render(<SocialIcons />);
+  expect(container.firstChild).toBeEmptyDOMElement();
+});
+
+it('renders social icons without links for own profile', () => {
+  const { getByTitle } = render(
+    <UserProfileContext.Provider value={{ isOwnProfile: true }}>
+      <SocialIcons />
+    </UserProfileContext.Provider>,
+  );
+  const element = getByTitle('Research Gate');
+  expect(element).toBeInTheDocument();
+  expect(element).not.toHaveAttribute('href');
```

Comment:
```suggestion
  expect(element.closest('a')).not.toHaveAttribute('href');
```
otherwise it doesn't say much

Reply:
Nice spot.